addons/l10n_dk_nemhandel/models/res_partner.py

```diff
@@ -179,7 +179,10 @@ def _check_nemhandel_participant_exists(self, participant_info, edi_identificati
 
         return edi_identification == participant_identifier and service_href.startswith(smp_nemhandel_url)
 
-    def _check_document_type_support(self, participant_info, ubl_cii_format):
+    def _check_document_type_support(self, participant_info, ubl_cii_format, process_type='billing'):
+        if self.country_code != 'DK':
```

Comment:
```suggestion
        if self._deduce_country_code() != 'DK':
```
Let's use the dedicated helper 🤓

Reply:
I don't think we need a separate PR, we can handle the change during the FW-port :relaxed: